### ISSUE TITLE
AttributeError: 'function' object has no attribute 'can_save'

### ISSUE THREAD

**Issue by ignatenkobrain** [mne-tools/mne-python#2582]
``` python
======================================================================
ERROR: Test CombineMarkersModel Traits Model
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/mne-python-0.10/mne/utils.py", line 633, in dec
    return function(*args, **kwargs)
  File "/builddir/build/BUILD/mne-python-0.10/mne/gui/tests/test_marker_gui.py", line 33, in test_combine_markers_model
    assert_false(model.mrk3.can_save)
AttributeError: 'function' object has no attribute 'can_save'
```

**Comment by agramfort** [comment]
how did you come up with this error? coregistration GUI?

**Comment by ignatenkobrain** [comment]
@agramfort I just started nosetests when packaged mne-python.

**Comment by agramfort** [comment]
certainly a conflict of traits versions. I am running 4.4.0

**Comment by ignatenkobrain** [comment]
I'm running 4.5.0 of Traits.

**Comment by agramfort** [comment]
that's weird.

can_save attribute is defined in MarkerPoints class.

no idea what's happening.

**Comment by Eric89GXL** [comment]
I think this has been fixed (hopefully) through the trail of #2648
